src/main/java/uk/gov/companieshouse/api/accounts/AccountsType.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.companieshouse.api.accounts;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * An enumeration of account types
+ */
+public enum AccountsType {
+
+    SMALL_FULL_ACCOUNTS("small-full", "small-full-accounts.html", "accounts#smallfull", "small_full_accounts", "small-full-accounts"),
+    ABRIDGED_ACCOUNTS("abridged", "abridged-accounts.html", "accounts#abridged", "abridged_accounts", "abridged-accounts");
+
+    private static final String ASSET_ID = "accounts";
+
+    private String accountType;
+    private String templateName;
+    private String kind;
+    private String resourceKey;
+    private String filingDescriptionKey;
+
+    AccountsType(String accountType, String templateName, String kind, String resourceKey,
+        String filingDescriptionKey) {
+        this.accountType = accountType;
+        this.templateName = templateName;
+        this.kind = kind;
+        this.resourceKey = resourceKey;
+        this.filingDescriptionKey = filingDescriptionKey;
+    }
+
+    public static AccountsType getAccountsType(String accountType) {
+        for (AccountsType accountsType : AccountsType.values()) {
+            if (accountsType.getAccountType().equals(accountType)) {
+                return accountsType;
+            }
+        }
+
+        return null;
```

Comment:
Is this exceptional behaviour? If so, a checked exception should be thrown here (which could become part of the stream by using an `orElseThrow` clause)